src/ChipArrayField/ChipArrayField.jsx

```diff
@@ -0,0 +1,209 @@
+import Chip from '@material-ui/core/Chip'
+import ChipInput from 'material-ui-chip-input'
+import PropTypes from 'prop-types'
+import React from 'react'
+import withStyles from '@material-ui/core/styles/withStyles'
+import { blue, grey } from '@material-ui/core/colors'
+
+const styles = theme => {
+  const light = theme.palette.type === 'light'
+  const borderColor = light ? 'rgba(0, 0, 0, 0.42)' : 'rgba(255, 255, 255, 0.7)'
+
+  return {
+    root: {
+      position: 'relative',
+      borderRadius: 4,
+      fontSize: 16,
+      backgroundColor: theme.palette.common.white,
+      border: `1px solid ${borderColor}`,
+      width: 'auto',
+      transition: theme.transitions.create('border-color', {
+        duration: theme.transitions.duration.shorter
+      }),
+      '&:focus': {
+        borderRadius: 4,
+        borderColor: '#80bdff',
+        boxShadow: '0 0 0 0 rgba(255,255,255,.25)'
+      },
+      '&:hover': {
+        borderRadius: 4,
+        borderColor: '#80bdff',
+        boxShadow: '0 0 0 0 rgba(255,255,255,.25)'
+      },
+      marginTop: '24px'
+    },
+    error: {
+      borderColor: theme.palette.error.main
+    },
+    input: {
+      marginTop: 4,
+      padding: '10px 12px'
+    },
+    chip: {
+      margin: '8px'
+    },
+    label: {
+      top: -12
+    }
+  }
+}
```

Comment:
I see that the border below the label input loose its colour. Is this the intended behaviour?

![border-without-color](https://user-images.githubusercontent.com/1538066/52674363-7f9b2b00-2f77-11e9-9f3e-e7fed9114e6c.gif)

Reply:
Nope, styling this has been a bit more of a pain than expected.

Reply:
After digging into this further, it's not something that can be styled through material-ui-chip-input, the solution is to put a pull request on that project, which will take time.

Depends how important we think it is.

Reply:
Since it's too complicated for a small stylistic choice, going to mark this as resolved!